### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by GiantRobots**
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Review by stevekinney — APPROVED** [review]

File: src/routes/login/index.svelte [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  <a href="/" class="flex my-4 max-h-8">
+    <img src="/logo.svg" alt="Temporal Logo" class="max-h-8" />
+  </a>
+</header>
+
+<section class="text-center my-[20vh]">
+  <h1 class="text-8xl font-semibold">Welcome back.</h1>
+  <p class="my-7">Lets get you signed in.</p>
+  <div class="mx-auto">
+    <Button
```
Comment (stevekinney) [inline src/routes/login/index.svelte:45]: This should just be a link that looks like a button, right?

File: src/routes/_header.svelte [hunk 2]
```diff
@@ -68,7 +58,11 @@
           />
         </button>
       {:else}
-        <button class="header-button" on:click={() => goto(loginUrl)}>
+        <button
```
[the remark this answers is not in the document]
Comment (GiantRobots) [reply src/routes/_header.svelte:61]: Actually this is super annoying to program as a link (runtime, settings funness) however we should have already redirected to the login page so Imma just drop this like it's hot.

File: src/lib/utilities/get-login-url.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+export const getLoginUrl = (
+  settings: Settings,
+  currentSearchParams: URLSearchParams,
+): string => {
+  const login = new URL('/auth/sso', settings.baseUrl);
+  const opts = settings.auth.options ?? [];
+
+  opts.forEach((option) => {
```
Comment (feedmeapples) [inline src/lib/utilities/get-login-url.ts:8]: after testing seems opts are not being added when making a request to `/auth/sso`

**Review by feedmeapples — CHANGES_REQUESTED** [review]

File: src/lib/utilities/get-login-url.ts (hunk 3 above)
Comment (feedmeapples) [reply src/lib/utilities/get-login-url.ts:8]: fixed in the last commit

**Review by feedmeapples — APPROVED** [review]

**Review by stevekinney — APPROVED** [review]

**Pull request merged**